spec/controllers/users/two_factor_authentication_controller_spec.rb

```diff
@@ -553,24 +596,32 @@ def index
     end
 
     context 'phone is not confirmed' do
+      let(:unconfirmed_phone) { '+1 (202) 555-1213' }
+      let(:parsed_phone) { Phonelib.parse(unconfirmed_phone) }
       before do
         @user = create(:user)
```

Comment:
i'd really like to refactor these tests so that they are using `lets` rather than instantiating an instance variable, but one of them was failing when i just fixed it, and i don't have time to dig around and try to figure out why. (i'm putting this on a list of things to fix eventually.)